src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Metadata/Controllers/PackagesController.cs

```diff
@@ -29,7 +29,7 @@ public PackagesController(IPackageService packageService)
     /// </summary>
     /// <param name="term">Søketerm.</param>
     /// <returns>Liste over søkeresultater.</returns>
-    [Route("search/{term}")]
+    [Route("search")]
     [HttpGet]
     public async Task<IEnumerable<SearchObject<PackageDto>>> Search(string term)
```

Comment:
```suggestion
    public async Task<IEnumerable<SearchObject<PackageDto>>> Search([FromQuery] string term)
```
I don't know if this is necessary or if maybe you just don't tend to use these here in the backend, but we use them in the BFF and personally I think it helps to make it more readable where the parameter comes from.

This is just a suggestion though, so if it works without and you have different conventions in backend, just resolve this comment 😊